src/SFA.DAS.AANHub.Api/Models/CreateApprenticeModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using SFA.DAS.AANHub.Application.Apprentices.Commands;
+
+namespace SFA.DAS.AANHub.Api.Models
+{
+    public class CreateApprenticeModel
+    {
+        public long ApprenticeId { get; set; }
+        public DateTime Joined { get; set; }
+        public int? Region { get; set; }
+        public string? Information { get; set; }
+        public string Email { get; set; } = null!;
+        public string Name { get; set; } = null!;
+
+        public static implicit operator CreateApprenticeMemberCommand(CreateApprenticeModel model) => new CreateApprenticeMemberCommand()
```

Comment:
Add coverage to check the implicit operator

Reply:
@MarkFCain, if you don't mind, can you point me to an example in RoATP as I haven't done this before. I am pretty sure I will be going in the wrong direction if I were to google this.

Reply:
@MarkFCain, just about to DM you on Slack about this. I may have a possible solution and hope it is what you are looking for

Reply:
Added test [here](https://github.com/SkillsFundingAgency/das-aan-hub-api/commit/27896da38874e783783dd7f45e35229895d16eb1#diff-9921de4e34f274206bc7db64567921a0fe3f47b594dc732ab332cc3318465d83)